invenio_app_ils/config.py

```diff
@@ -174,15 +174,15 @@ def _(x):
 # TODO: uncomment when this PR:
 # https://github.com/inveniosoftware/invenio-app/pull/32
 # is merged and released and if u want to work on UI
-# APP_DEFAULT_SECURE_HEADERS = {"content_security_policy": {}}
+APP_DEFAULT_SECURE_HEADERS = {"content_security_policy": {}}
 # OAI-PMH
 # =======
 OAISERVER_ID_PREFIX = "oai:invenio_app_ils.com:"
 
 ###############################################################################
 # Debug
 ###############################################################################
-DEBUG = True
+DEBUG = False
```

Comment:
revert

Reply:
Food for thought, we can have the debug value deriving from an env value, so its `False` when prod but `True` when dev.

Reply:
Until now in prod we have a `.cfg` file that overrides all these.